executor/execute.go

```diff
@@ -0,0 +1,108 @@
+package executor
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/blocklessnetworking/b7s/models/execute"
+)
+
+// execute handles the actual execution of the Blockless function. It returns the
+// standard output of the blockless-cli that handled the execution. `Function`
+// typically takes this output and uses it to create the appropriate execution response.
+func (e *Executor) execute(requestID string, req execute.Request) (string, error) {
+
+	e.log.Info().
+		Str("id", req.FunctionID).
+		Str("request_id", requestID).
+		Msg("processing execution request")
+
+	// Generate paths for execution request.
+	paths := e.generateRequestPaths(requestID, req.FunctionID, req.Method)
+
+	err := os.MkdirAll(paths.workdir, defaultPermissions)
+	if err != nil {
+		return "", fmt.Errorf("could not setup working directory for execution (dir: %s): %w", paths.workdir, err)
+	}
+	// Remove all temporary files after we're done.
+	defer func() {
+		err := os.RemoveAll(paths.workdir)
+		if err != nil {
+			e.log.Error().Err(err).Str("dir", paths.workdir).
+				Msg("could not remove request working directory")
+		}
+	}()
+
+	e.log.Debug().
+		Str("dir", paths.workdir).
+		Str("request_id", requestID).
+		Msg("working directory for the request")
+
+	// TODO: Super hackish, but ported. See why this is actually needed.
```

Comment:
the runtime currently doesn't operate in memory, so we need to feed it an execution manifest. This should change soon as we continue to smooth out this operating process.